src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -762,6 +842,30 @@ public int deleteMarkedForDelete(String origin) throws Exception {
     }
 
     
+    /**
+     * 
+     * Update a mapping new a new kalturaId
+     * 
+     * @param mappingDto
```

Comment:
Small typo in the first line ' new a new'. Also, please either add a description for the mappingDto or remove it from the JavaDoc.

Reply:
fixed